DeDRM_plugin/config.py

```diff
@@ -154,7 +154,7 @@ def get_help_file_resource():
             # link is clicked in case the helpfile is updated in newer plugins.
             file_path = os.path.join(config_dir, "plugins", "DeDRM", "help", help_file_name)
             with open(file_path,'w') as f:
-                f.write(self.load_resource(help_file_name))
+                f.write(self.load_resource(help_file_name).decode())
```

Comment:
Wouldn't it be better to open `file_path` in binary mode and directly write the resource bytes instead of round-tripping through a string?

Reply:
good point. ill give yhat a try shortly.